app/models/post.rb

```diff
@@ -1,17 +1,22 @@
-class Post < ApplicationRecord
-  after_save :update_posts_counter
-
-  belongs_to :author
-  has_many :comments
-  has_many :likes
-
-  def five_recent_comments
-    comments.last(5)
-  end
-
-  private
-
-  def update_posts_counter
-    author.update(posts_counter: author.posts.length)
-  end
-end
+class Post < ApplicationRecord
+  after_save :update_posts_counter
+
+  belongs_to :author
+  has_many :comments
+  has_many :likes
+
+  validates :title, presence: true
+  validates :title, length: { maximum: 250 }

```

Comment:
- Optional
You can combine these two validations into one. Like this ⬇️
`validates : title, presence: true, length: {maximum: 250}`